### PR TITLE
I added the queries

### PR DESCRIPTION
**Pull request by EddyShimwa**
## On this task,
- I used correct Gitflow.
- I documented my work in a professional way.
- I Copied the SQL that I used into the required files.

### Screenshots,
- Task 1:  Find all animals whose name ends in "mon".

image

- Task 2:  List the name of all animals born between 2016 and 2019.

image

- Task 3 List the name of all animals that are neutered and have less than 3 escape attempts.

image

- Task 4 List the date of birth of all animals named either "Agumon" or "Pikachu"

image

- Task 5 List name and escape attempts of animals that weigh more than 10.5kg

image

- Task 6 Find all animals that are neutered.

image

- Task 7 Find all animals not named Gabumon.

image

- Task 8 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

image

### CODE REVIEW

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @EddyShimwa  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### Please ensure the MIT License link you provided is valid. It's currently broken 👍🏽
Screenshot from 2023-02-22 19-07-50
- [ OPTIONAL ] Kindly fill out the FAQ section of your README file to make it more professional. 😉 
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by EddyShimwa** [comment]
Thanks @rotimiazeez for reviewing this PR kindly! Let me make the changes you suggested!

File: README.md [hunk 1]
```diff
@@ -63,9 +66,9 @@
 
 > Describe between 1-3 key features of the application.
```
Comment (OyePriscilla) [inline README.md:68]: - [optional] Please kindly remove all the placeholders in your README file or you customize them as appropriate. The placeholders are there to describe what to do with each header. 👍

**Review by OyePriscilla — APPROVED** [review]
Hi @EddyShimwa 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Added MIT License Link ✔️ 
PR title and summary professional ✔️ 

## Optional suggestions

- [optional] Kindly remove the FAQ header from the REAME file if you have no questions to answer, this will make your repo look more professional. Thanks. 🙏  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (EddyShimwa) [reply README.md:68]: Thanks @OyePriscilla For reviewing this PR! I'll do it!

Comment (OyePriscilla) [reply README.md:68]: @EddyShimwa Great! You're welcome.

**Pull request merged**